ui/apps/platform/src/Containers/Collections/CollectionResults.tsx

```diff
@@ -248,8 +254,20 @@ function CollectionResults({
 
     return (
         <>
+            <div className="pf-u-p-lg pf-u-display-flex pf-u-align-items-center">
```

Comment:
Do we have any team preferences for when to use the PF `<Flex>` component versus `<div className="pf-u-display-flex">`? I know the former is required if we need to change the flex properties based on screen size, but it can be fairly verbose for simple flex cases like this.